### ISSUE TITLE
[Question] Firefox's total cookie protection

### ISSUE THREAD

**Issue by angeousta** [brave/brave-browser#31287]
Is there an equivalent of https://support.mozilla.org/en-US/kb/introducing-total-cookie-protection-standard-mode for brave?

**Comment by Emi-TheDhamphirInLoveUnderTheFrozenStar** [comment]
Yes, it is called Ephemeral Storage, which was implemented way before Firefox did their TCP.
It is already the default, that's why Brave doesn't 'block 3p cookies', they put them in the temporary storage and clear them as soon as you close the website.
In fact, if you go to Cookies settings, you will see the one that says `Site that clear cookies when you close them`, that uses the Ephemeral Storage for the first-party storages, so it will never touch the 'normal' storage, it will stay just the same and only place the new cookies or local storage items, etc etc, in the Ephemeral Storage and clear it when you close the website, both storages live, which means, you can 'switch' between them as long as you don't close the website and you just refresh the page.
